docs/book/component-guide/orchestrators/azureml.md

```diff
@@ -0,0 +1,143 @@
+for a shortcut on how to deploy & register this stack component.
+{% endhint %}
+
+In order to use an AzureML orchestrator, you need to first 
+deploy [ZenML to the cloud](../../getting-started/deploying-zenml/README.md). 
+It would be recommended to deploy ZenML in the same region as you plan on 
+using for AzureML, but it is not necessary to do so. You must ensure that 
+you are connected to the remote ZenML server before using this stack component.
+
+## How to use it
+
+In order to use the AzureML orchestrator, you need:
+
+* The ZenML `azure` integration installed. If you haven't done so, run:
+
+```shell
+zenml integration install azure
+```
+
+* [Docker](https://www.docker.com) installed and running.
+* A [remote artifact store](../artifact-stores/artifact-stores.md) as part of your stack.
+* A [remote container registry](../container-registries/container-registries.md) as part of your stack.
+* An [Azure resource group equipped with an AzureML workspace](https://learn.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources?view=azureml-api-2) to run your pipeline on.
+
+There are two ways of authenticating your orchestrator with AzureML:
+
+1. **Default Authentication** simplifies the authentication process while 
+developing your workflows that deploy to Azure by combining credentials used in 
+Azure hosting environments and credentials used in local development.
+2. **Service Principal Authentication (recommended)** is using the concept 
+of service principals on Azure to allow you to connect your cloud components 
+with proper authentication. For this method, you will need to [create a service 
+principal on Azure](https://learn.microsoft.com/en-us/azure/developer/python/sdk/authentication-on-premises-apps?tabs=azure-portal), 
+assign it the correct permissions and use it to [register a ZenML Azure Service 
+Connector](https://docs.zenml.io/how-to/auth-management/azure-service-connector).
+    ```bash
+    zenml service-connector register <CONNECTOR_NAME> --type azure -i
+    zenml orchestrator connect <ORCHESTRATOR_NAME> -c <CONNECTOR_NAME>
+    ```
+
+## Docker
+
+For each pipeline run, ZenML will build a Docker image called 
+`<CONTAINER_REGISTRY_URI>/zenml:<PIPELINE_NAME>` which includes your code 
+and use it to run your pipeline steps in AzureML. Check out 
+[this page](../../how-to/customize-docker-builds/README.md) if you want to 
+learn more about how ZenML builds these images and how you can customize them.
+
+## AzureML UI
+
+Each AzureML workspace comes equipped with an Azure Machine Learning studio. 
+Here you can inspect, manage, and debug your pipelines and steps.
+
+![AzureML pipeline example](../../.gitbook/assets/azureml-pipelines.png)
+
+Double-clicking any of the steps on this view will open up the overview page 
+for that specific step. Here you can check the configuration of the component 
+and its execution logs.
+
+## Settings
+
+The ZenML AzureML orchestrator comes with a dedicated class called 
```

Comment:
can you give a concrete code example of how to set this and pass to a pipeline?

Reply:
I added an example.